### PR TITLE
Fix some test labels

### PR DESCRIPTION
**Pull request by infiniteregrets**
- operator tests were running verify-config tests, which are not needed there
- 1337 tests were not decorated
- ignoring one of the operator tests pending a fix

### CODE REVIEW

**Review by aviramha — APPROVED** [review]
Great!

File: tests/src/operator.rs [hunk 1]
```diff
@@ -166,6 +166,7 @@ mod operator {
         assert!(!res.success());
     }
 
+    #[ignore]
```
Comment (aviramha) [inline tests/src/operator.rs:169]: maybe add a comment

**Review by aviramha — APPROVED** [review]

**Review by aviramha — COMMENTED** [review]
Nice, thank you.

**Comment by aviramha** [comment]
Wait, maybe we can split into directory hierarchy to make it easier to follow?
Put all cli tests in cli dir, then `use` the cli dir module with cfg attr, reducing the need to specify for each test?

**Comment by infiniteregrets** [comment]
> Wait, maybe we can split into directory hierarchy to make it easier to follow? Put all cli tests in cli dir, then `use` the cli dir module with cfg attr, reducing the need to specify for each test?

done

**Pull request merged**